forge/test/operators/pytorch/nn/test_convtranspose2d.py

```
@@ -0,0 +1,323 @@
+        input_params: List[Dict] = [],
+        number_of_operands: int = 1,
+        warm_reset: bool = False,
+    ):
+        """Common verification function for all tests"""
+
+        input_source_flag: InputSourceFlags = None
+        if test_vector.input_source in (InputSource.FROM_DRAM_QUEUE,):
+            input_source_flag = InputSourceFlags.FROM_DRAM
+
+        operator = getattr(torch.nn, test_vector.operator)
+
+        kwargs = test_vector.kwargs if test_vector.kwargs else {}
+
+        model_type = cls.MODEL_TYPES[test_vector.input_source]
+        if test_vector.input_source == InputSource.CONST_EVAL_PASS:
+            pytorch_model = model_type(
+                operator=operator,
+                opname=test_vector.operator,
+                shape=test_vector.input_shape,
+                kwargs=kwargs,
+                dtype=TestTensorsUtils.get_dtype_for_df(test_vector.dev_data_format),
+            )
+        else:
+            pytorch_model = model_type(
+                operator=operator,
+                opname=test_vector.operator,
+                shape=test_vector.input_shape,
+                kwargs=kwargs,
+            )
+
+        input_shapes = tuple([test_vector.input_shape for _ in range(number_of_operands)])
+        logger.trace(f"***input_shapes: {input_shapes}")
+
+        VerifyUtils.verify(
+            model=pytorch_model,
+            test_device=test_device,
+            input_shapes=input_shapes,
+            input_params=input_params,
+            input_source_flag=input_source_flag,
+            dev_data_format=test_vector.dev_data_format,
+            math_fidelity=test_vector.math_fidelity,
+            pcc=test_vector.pcc,
+            warm_reset=warm_reset,
+            deprecated_verification=False,
+            verify_config=VerifyConfig(value_checker=AllCloseValueChecker(rtol=1e-2, atol=1e-2)),
+            value_range=ValueRanges.SMALL,
+            skip_forge_verification = True,
```

We agreed to skip test execution until implementation and failing rules are ready instead of just skipping Forge verification.